packages/react-dom/src/__tests__/ReactUpdates-test.js

```diff
@@ -886,6 +886,11 @@ describe('ReactUpdates', () => {
       'Invalid argument passed as callback. Expected a function. Instead ' +
         'received: [object Object]',
     );
+    component = ReactTestUtils.renderIntoDocument(<A />);
```

Comment:
Please add a comment like `// Make sure the warning is deduplicated and doesn't fire again` before these new lines here and below. Otherwise it's not obvious what's being tested.